_posts/2024-02-18-how-its-made.md

```diff
@@ -0,0 +1,185 @@
+The binary is then combined with various other data files to create the game you can download from our website or for example get via Steam.
+
+While every computer program has a source, not every program is Open Source.
+Open Source requires that the source code is freely available and allows modifications by other parties.
+You can look at the source code of OpenTTD right now in our [git repository on GitHub](https://github.com/OpenTTD/OpenTTD) <small>(but please come back here afterwards)</small>.
+To be able to track changes to the source code, we use a version control system called <code>git</code>.
+It shows us each and every individual change (called commit in git) that was made to the game since its inception 20 years ago.
+Being able to see old changes is quite useful when trying to fix problems or figuring out why something was done the way it was done.
+The list of commits is also what you need to watch to gain "magical" knowledge about things that are coming in future releases of OpenTTD.
+
+![List of commits on GitHub]({% link /static/img/post_2024-02-18-how-its-made/commits.png %}){:width="100%"}
+
+Our GitHub page is also where things are tracked that are not yet part of the game and still in development.
+You might have encountered it before as the place where you can report an issue with the game.
+Besides the git repository of the game itself, GitHub also hosts various other git repository related to the game, like for example OpenGFX, NewGRF development tools or various back-end infrastructure.
+
+## How a feature is born
+
+Now that we know *where* development happens, let's look at *how* it happens.
+
+Each and every change to OpenTTD, no matter if it is a large new feature or just a simple one-line bug fix, starts with someone choosing to spend their time on it.
+As the source code of the game is freely accessible, this can be literally anyone, for a multitude of reasons.
+
+Many features come about because someone has an idea how their game could be better and decides to work on it.
+Bugs are usually fixed because someone either feels responsible for it in some way or is just plain annoyed by it.
+But really, everybody has a different personal motivation.
+There's one reason you're not going to find though: Because the boss said so, or because it is on the big company road map.
+OpenTTD is purely developed by volunteers, there's no company, foundation, or other controlling central entity behind it.
+
+Now let's assume that the person is in fact developing a feature (like for example the new ship pathfinder) and has completed the initial development.
+Since OpenTTD moved to its current home on GitHub almost 6 years ago, we've adopted a fixed process for any change to OpenTTD.
+The first step in the process is to open a so-called pull request (PR) on GitHub with the proposed changes to the source code in the form of git commits.
```

Comment:
In general, I don't like the phrasing "so-called", it seems to me to indicate that what it's describing is a "bad" or "unsuited" term in some way. Maybe instead, "what is called a *pull request* (PR)".